lstchain/calib/camera/calibration_calculator.py

```diff
@@ -210,10 +210,12 @@ def calculate_calibration_coefficients(self, event):
         # define unusables on number of estimated pe
         npe_deviation =  calib_data.n_pe - npe_median[:,np.newaxis]
 
-        # cut on the base of pe statistical uncertanty (neglect the 7% spread due to detection QE)
+        # cut on the base of pe statistical uncertainty (adding a 7% spread due to different detection QE among PMs) 
+        tot_std = np.sqrt(npe_median + (0.07 * npe_median)**2)
```

Comment:
Should this be configurable? i.e. `camera_qe_std_percent = FloatTelescopeParameter(0.07).tag(config=True)`

Alternatively, use a module constant.

Reply:
mah, I was also wondering... ok, let's do it configurable

Reply:
done